scripts/postprocessing/coverage_analysis/readme.md

```diff
@@ -0,0 +1,134 @@
+# **COVERAGE ANALYSIS README**
```

Comment:
So glad you changed this to Markdown! I considered doing that, but didn't want to become responsible for maintaining it :)

Reply:
Did this come from Andres' pr?

Reply:
I think so. I didn't make any changes to coverage_analysis